aspnetcore-features/CheckboxListSolution/CheckboxList/Controllers/HomeController.cs

```diff
@@ -0,0 +1,47 @@
+﻿using CheckboxList.Data;
+using CheckboxList.Models;
+using Microsoft.AspNetCore.Mvc;
+using System.Diagnostics;
+
+namespace CheckboxList.Controllers
+{
+    public class HomeController : Controller
+    {
+        public IActionResult Index()
+        {
+            var model = new InitialModel();
+            return View(model);
+        }
+
+        [HttpPost]
+        public IActionResult Index(InitialModel model)
+        {
+            return RedirectToAction("Index");
+        }
+
+        public IActionResult CourseSelection()
+        {
+            var model = Repository.GetCourses();
+            return View(model);
+        }
+
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public IActionResult CourseSelection(List<CheckboxViewModel> courses)
+        {
+            var selectedCourses = courses.Where(x => x.IsChecked == true).ToList();
```

Comment:
We don't need to be explicit in the `Where` clause with the `== true`. This can simply be `Where(x => x.IsChecked)`

Reply:
updated the code.